### PR TITLE
Integrate cookie-policy v3.0.4

### PR DESCRIPTION
**Pull request by anthonydillon**
## Done
Integrate cookie-policy v3.0.4

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8041
- Check the cookie policy appears
- Check the manager opens and works
- Make a selection and the notification should disappear
- Scroll to the footer and click the Manage link in the footer
- See that manager appears again.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -6,7 +6,9 @@
     "watch": "watch -p 'static/sass/**/*.scss' -c 'yarn run build'",
     "build": "yarn run build-css && yarn run build-js",
     "build-css": "node-sass --include-path node_modules static/sass --source-map true --output-style compressed --output static/css && postcss --map false --use autoprefixer --replace 'static/css/**/*.css'",
-    "build-js": "mkdir -p static/js/build/global-nav && cp node_modules/@canonical/global-nav/dist/global-nav.js static/js/build/global-nav",
+    "build-js": "yarn run copy-cookie-policy && yarn run copy-global-nav",
+    "copy-cookie-policy": "mkdir -p static/js/modules/cookie-policy && cp node_modules/@canonical/cookie-policy/build/js/cookie-policy.js static/js/modules/cookie-policy",
```
Comment (pmahnke) [inline package.json:10]: I think you are confusing `build` with `modules`

**Review by pmahnke — CHANGES_REQUESTED** [review]
I think the global nav and cookies are in different directories. Also, can you change the GTM code to `GTM-K9KCMZ`

**Comment by pmahnke** [comment]
oh, I thought this was for jaas.ai... sorry @anthonydillon

**Comment by anthonydillon** [comment]
Superseded by #151

**Pull request closed without merging**